### ISSUE TITLE
Review SEO of docs

### ISSUE THREAD

**Issue by astrojuanlu** [readthedocs/readthedocs.org#8537]
As discussed in https://github.com/readthedocs/readthedocs.org/pull/8528, among all the overhaul tasks we could do on our docs, reviewing its SEO is one of them. Things like:

- Improving page descriptions using metadata
- Matching titles with URLs
- Focus on specific keywords for searchability
- Use of structured metadata
- ...?

I have little experience with this, but I guess there are some automated tools from Google and others that can be of help.

**Comment by ericholscher** [comment]
Luckily we have a guide on this :) https://docs.readthedocs.io/en/latest/guides/technical-docs-seo-guide.html

**Comment by ericholscher** [comment]
@humitos I think you mentioned this other day, already have an issue for it :)

**Comment by humitos** [comment]
Yes. I mentioned specifically the usage of OpenGraph's `meta` description. We can use this extension for that https://sphinxext-opengraph.readthedocs.io/en/latest/

**Comment by ericholscher** [comment]
Agreed 👍
